packages/@lwc/babel-plugin-component/package.json

```diff
@@ -14,8 +14,8 @@
         "src/post-process/"
     ],
     "dependencies": {
-        "@babel/helper-module-imports": "7.0.0",
-        "@babel/plugin-proposal-class-properties": "7.1.0",
+        "@babel/helper-module-imports": "7.8.3",
```

Comment:
We should upgrade all the Babel related packages at the same time. I have been reluctant to do that so far because we have some typings issues with the LWC template compiler, when bumping Babel version. Let's roll those upgrades back.